pkg/dbutil/common.go

```diff
@@ -756,3 +756,22 @@ func ignoreDDLError(err error) bool {
 		return false
 	}
 }
+
+// DeleteRows delete rows in several times. Only can delete less than 300,000 one time in TiDB.
+func DeleteRows(ctx context.Context, db *sql.DB, schemaName string, tableName string, where string, args []interface{}) error {
```

Comment:
why not just truncate table

Reply:
TRUNCATE doesn't accept a `where` clause